runtime/doc/api.txt

```diff
@@ -2335,7 +2335,8 @@ nvim_buf_set_extmark({buffer}, {ns_id}, {line}, {col}, {opts})
                                 Defaults to false.
                               • priority: a priority value for the highlight
                                 group. For example treesitter highlighting
```

Comment:
while changing this, could be worth mentioning that the API-level default is 4096 (otherwise the quick reader will assume that is 100 as well).

Reply:
Maybe it would be good to expose those as Lua constants so that you can say `EXTMARK_DEFAULT_PRIORITY + 50`, `EXTMARK_TREESITTER_PRIORITY + 50`